packages/docs/src/styles/components/footer.scss

```diff
@@ -32,6 +32,12 @@
 }
 
 .c-footer__content--bottom {
+  padding-bottom: $spacer-7 + $spacer-2;
```

Comment:
Wonder if I should add more spacer values to tokens :thinking:

Reply:
Not sure - this looks like a 1-off thing in our designs, so I don't know if it warrants tokenizing (or using the spacer vars at all)